components/webac/src/main/java/org/trellisldp/webac/WebACService.java

```diff
@@ -273,6 +273,18 @@ private static IRI cleanIdentifier(final IRI identifier) {
         return rdf.createIRI(cleanIdentifier(identifier.getIRIString()));
     }
 
+    /**
+     * Get the structural-logical container for this resource.
+     *
+     * @param identifier the resource identifier
+     * @return a container, if one exists. Only the root resource would return empty here.
+     */
+    private static Optional<IRI> getContainer(final IRI identifier) {
```

Comment:
This looks like the default impl of `getContainer`, no? If so, should we factor this out somewhere outside of WebAC for potential use elsewhere, or am I being paranoid...?

Reply:
It is _exactly_ the default impl of `getContainer`. One possibility is to add this as a method in `org.trellisldp.api.RDFUtils`. In fact, if you like that idea, I'll do just that.

Reply:
Hm. I was thinking that `RDFUtils` is more generic, more like stuff usable on _any_ RDF, but something called `BlahUtils` pretty much by name alone does not have very precise semantics. 😁 

Would you be amenable to introducing a `LDPUtils` sister class to `RDFUtils`? Or is that more annoying because people will have to find it...

Reply:
I agree that `RDFUtils` is something of a misnomer, and not just because of this method. I was thinking of renaming it to `TrellisUtils`.

Reply:
That would be better than introducing a new `BlahUtils`. For the mo, moving the method there is good enough for me!